### PR TITLE
Make loading tips more readable

### PR DESCRIPTION
**Pull request by Crotalus**
Add dropshadow so the loading tips text gets more constrast compared to the background

### CODE REVIEW

File: lua/ui/game/gamemain.lua [hunk 1]
```diff
@@ -385,6 +386,7 @@ function CreateWldUIProvider()
         local text = '::  ' .. LOC('<LOC LOAD_0000>IN TRANSIT') .. '  ::'
         local textControl = UIUtil.CreateText(background, text, 20, UIUtil.bodyFont)
         textControl:SetColor(color)
+        textControl:SetDropShadow(true)
```
Comment (speed2CZ) [inline lua/ui/game/gamemain.lua]: This one just flashes for a split of the second when the loading is done

Comment (Crotalus) [inline lua/ui/game/gamemain.lua]: Good catch, added it via github's web editor. Should probably be squashed into one commit (not at home atm)

**Comment by micheljung** [comment]
And are they still flashing? If so, I think that should be removed, too :)

**Comment by HUSSAR-mtrela** [comment]
why dont you use colors from UIUtil which are also used in existing UI elements?
```
textControl:SetColor(UIUtil.bodyColor) 
textControl:SetColor(UIUtil.tooltipTitleColor) 
textControl:SetColor(UIUtil.fontColor) 
```

**Comment by speed2CZ** [comment]
@micheljung well IN TRANSIT is flashing
so I made the tips flash as well cuz it looked bad if one was flashing and second didnt.
Maybe slow donw your PC and you'll have more than enough time to read it even backwards!

**Pull request merged**